Update hello world tutorial - Closes #858 (#861)
LiskHQ/lisk-docs: modules/ROOT/pages/tutorials/hello-world.adoc

After creating the application instance, it is possible to register custom modules and plugins with the application.
The implementation of the `HelloModule` and the `HelloAPIPlugin` is explained below.

> `HelloAPIPlugin` is a default plugins which comes with the Lisk SDK.

Reviewer: ```suggestion
`HelloAPIPlugin` is a default plugin that comes with the Lisk SDK.
```

=== The Hello API plugin

> In addition to the hello module, we add a `HelloAPIPlugin`, which provides additional API endpoints.

Reviewer: ```suggestion
In addition to the hello module, a `HelloAPIPlugin` is added which provides additional API endpoints.
```

> These API endpoints allow the react client get information about the newly created hello properties of the application via API calls.

Reviewer: ```suggestion
These API endpoints allow the react client to get information about the newly created hello properties of the application via API calls.
```
